### PR TITLE
[16.0][MIG] account_analytic_parent: Migration to 16.0

### PR DESCRIPTION
**Pull request by mohammadazeem786**

### CODE REVIEW

**Comment by StefanRijnhart** [comment]
Please amend your pre-commit commit. The module name is missing in its description:
image

**Comment by StefanRijnhart** [comment]
Functional question: now that you can link an analytic plan to analytic accounts in Odoo 16, and analytic plans have a hierarchical parent structure, do we still need this module? @mylbco 

image

**Comment by StefanRijnhart** [comment]
To answer my own question, adding plans adds an entry to the analytic distribution widget which is not what we want for the analytic parent structure, so this module is *not* made obsolete by the introduction of analytic plans in Odoo 16 (IMHO).
image

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -113,7 +113,7 @@ repos:
           - requirements.txt
           - --header
           - "# generated from manifests external_dependencies"
-  - repo: https://gitlab.com/PyCQA/flake8
+  - repo: https://github.com/PyCQA/flake8
```
Comment (StefanRijnhart) [inline .pre-commit-config.yaml]: Fixed properly in https://github.com/OCA/account-analytic/pull/502. Please rebase.

**Review by StefanRijnhart — APPROVED** [review]
~~Thanks! Code reviewed + tested.~~

**Comment by StefanRijnhart** [comment]
Oh, one more thing: the parents are not visible in the new analytic distribution widget, which makes it impossible to know which is which. Can you check this? 
image

**Comment by rafaelbn** [comment]
I cannot test runboat

imagen

**Comment by StefanRijnhart** [comment]
Strange to find runboat not working. My screenshots were all from runboat.

**Comment by rafaelbn** [comment]
Hello @sbidoul , please do you know hoy to detect the reason of this runboat not working? Thank you! 🙏 ❤️

**Comment by sbidoul** [comment]
Hm. Expired microk8s API certificate. Weird. I'm AFK for a little while but found a temporary workaround, hopefully.

**Comment by HaraldPanten** [comment]
@rafaelbn ready to test 👍

**Comment by rafaelbn** [comment]
@StefanRijnhart would you like to finish your review? 😄

**Comment by StefanRijnhart** [comment]
It seems the issue from https://github.com/OCA/account-analytic/pull/501#issuecomment-1334925620 has been resolved!

image

**Review by StefanRijnhart — APPROVED** [review]

**Review by robinkeunen — APPROVED** [review]
LGTM 👌

**Pull request merged**